sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,219 @@
+### SYCL Library
+ - Specialized atomic `fetch_add`, `fetch_min` and `fetch_max` for
+   floating-point types [37a9a2a] [59ceaf4]
+ - Added support for accessors to array types [7ed4f58]
+ - Added sub-group information queries on CUDA [c36fa65]
+ - Added support for `sycl::queue::barrier` in Level Zero plugin [7c31f90]
+ - Improved runtime memory usage in Level Zero plugin [c9d71d4] [2ce2ca6]
+   [46e3c64]
+ - Added Level Zero interoperability with specifying of ownership [41221e2]
+ - Improved runtime memory usage when using USM [461fa02]
+ - Provided facility for user to control execution range rounding [f6ac45f]
+ - Ensured correct access mode in `sycl::handler::copy()` method [b489479]
+ - Disallowed for atomic accessors in `sycl::handler::copy()` method [14437db]
+ - Provided move-assignability of `usm_allocator` class [05a805e]
+ -  Improved performance of copying data during native memory object creation
+    on devices without host unified memory [ad8c9d1]
+ - [ESIMD] Added implicit set up of fence before barrier as required by hardware
+   [692228c]
+ - Allowed for using of interoperability program constructor with multi-device
+   context [c7f7674]
+ - Allowed trace of Level Zero calls only with `SYCL_PI_TRACE=-1` [ea73219]
+ - Added throw of `feature_not_supported` when when upon attempt to create
+   program using `create_program_with_source` with Level Zero or CUDA [ba77e3a]
+ - Added support for `inline` `cl` namespace in debugger [8e441d4]
+ - Added support for build with GCC 7 [d8fea22]
+ - Added in-memory caching of programs built with custom build options
+   [86b0e8d] [e152b0d]
+ - Improved range rounding heuristics [7efb692]
+ - Added `get_backend` methods to SYCL classes [ee7e99f]
+ - Added `sycl::sub_group::load` and `sycl::sub_group::store` versions that
+   take raw pointers [248f550]
+ - Enabled caching of devices in `sycl::device` interoperability constructors
+   [d3aeb4a]
+ - Added a warning on using SYCL 1.2.1 OpenCL interoperability API when
+   compiling in SYCL 2020 mode. It can be suppressed by defining
+   `SYCL2020_DISABLE_DEPRECATION_WARNINGS` macro or by setting `-sycl-std=2017`
+   or `-sycl-std=1.2.1` [a249316]
+ - Added support for blitter engine in Level Zero plugin. Some memory
+   operations are submitted to a Level Zero copy queue now [11ba5b5]
+ - Improved `sycl::INTEL::lsu::load` and `sycl::INTEL::lsu::store` to take
+   `sycl::multi_ptr` [697469f]
+ - Added a diagnostic on attempt to compile a SYCL application without dynamic
+   C++ RT on Windows [d4180f4]
+ - Added support for `Queue Order Properties` extension for Level Zero [50005c7]
+ - Improved plugin discovery mechanism - if a plugin fails to initialize others
+   will be discovered anyway [d513074]
+ - Added support for `sycl::info::partition_affinity_domain::numa` in Level
+   Zero plugin [2ba8e05]
+### Documentation
+ - Added notes on SYCL 2020 status and runtime headers C++14 standard
+   requirement [a05f5de] [fed2d47]
```

Comment:
```suggestion
 - Added notes on SYCL 2020 status and runtime headers C++17 standard
   requirement [a05f5de] [fed2d47]
```

Reply:
The commit which does this is not included in the release.

Reply:
I would just remove this item from the list to avoid confusion.